_data/CONTRIBUTORS.yaml

```diff
@@ -559,6 +559,16 @@ Gil Poiares-Oliveira:
     orcid: 0000-0003-4638-2879
     email: gil.oliveira@tecnico.ulisboa.pt
     affiliation: INESC-ID / BioData.pt / ELIXIR-PT
+Fotis Psomopoulos:
+    git: fpsom
+    orcid: 0000-0002-0222-4273
+    email: fpsom@certh.gr
+    affiliation: Institute of Applied Biosciences (INAB), at the Centre for Research and Technology Hellas (CERTH) / ELIXIR-GR 
```

Comment:
@sfragkoul could you shorten the affiliation? This might break the layout of the contributor tiles :)